src/ComputeSharp.D2D1/Attributes/Enums/D2D1PixelOption.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ComputeSharp.D2D1;
+
+/// <summary>
+/// Indicates how the sampling of a D2D1 pixel shader will be restricted. This indicates whether the vertex buffer
```

Comment:
The D2D doc for this is wrong and should not be copied as-is. Notice how it says "... the _vertex buffer_ ...". Ugh :) It also has nothing to do with whether the buffer is large and/or how often it changes

Reply:
Ugh, these docs... 😆
Fixed in ca05ea2a5fb90a27ffd91dcbd3f4326d321fe0a2, thanks!